### PR TITLE
follow tidyverse style guide

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

**Comment by ianmcook** [comment]
Thanks @IndrajeetPatil! This was done using `lintr::lint_package()`, correct?

99% of this looks great to me, but there are a couple of small ways that I think the default lintr settings make things harder to read. I'll leave a few comments about that here. If you have time to apply those changes in this PR and also to the lintr config in #20, that would be awesome. Thank you!

**Comment by IndrajeetPatil** [comment]
Hi @ianmcook! 

This was actually done using `styler` package. 

`lintr` github action will just flags the lints, but won't do anything about them. I will push the `.lintr` config file that you can modify to your liking. 

As for the stylistic changes you don't like, sure, please comment and I will revert the ones you don't like.

File: tests/testthat/setup-load_data.R [hunk 1]
```diff
@@ -4,30 +4,31 @@ suppressPackageStartupMessages(library(dbplyr))
...
+    db_con <<- DBI::dbConnect(RSQLite::SQLite(), ":memory:")
+    copy_to(db_con, iris, name = "iris_db")
+    iris_db <<- tbl(db_con, "iris_db")
 
-  db_con        <<- DBI::dbConnect(RSQLite::SQLite(), ":memory:")
-  copy_to(db_con, iris, name = "iris_db")
-  iris_db       <<- tbl(db_con, "iris_db")
-
-  iris_dt       <<- lazy_dt(iris)
-  flights_dt    <<- lazy_dt(flights)
```
Comment (ianmcook) [inline tests/testthat/setup-load_data.R]: Could you please leave these lines as is? Squashing the extra whitespace makes this part much less readable

File: R/query.R [hunk 2]
```diff
@@ -380,15 +376,10 @@ query_ <- function(data, sql, query = TRUE) {
...
     }
-
   } else {
-
     if (use_quoted_deparsed_expressions) {
-
       out <- out %>% verb(transmute, !!!(quote_full_expressions(final_select_list)))
-
     } else if (!transmute_early) {
-
```
Comment (ianmcook) [inline R/query.R]: Could you please preserve empty lines? The loss of vertical whitespace makes some sections of this code harder for me to read

**Comment by IndrajeetPatil** [comment]
I have reverted the two changes you did not like. Let me know if you want me to change something else.

**Pull request merged**